tests/acceptance/new-test.js

```diff
@@ -276,20 +277,24 @@ describe('Acceptance: ember new', function () {
     expect(file('.named-ref')).to.exist;
   });
 
-  it('ember new with shorthand git blueprint and ref checks out the blueprint with the correct ref and uses it', async function () {
-    this.timeout(20000); // relies on GH network stuff
+  // Temporarily skipped for Node versions that ship with npm v6 by default.
+  // See https://github.com/npm/cli/issues/4896 for more info.
+  if (semver.gte(process.version, 'v16.0.0')) {
```

Comment:
As far as I can tell from reading that issue, the issue is with `npm@6` and is not a problem with `npm >=7`, right?

If that is the case, can we check for the `npm` version instead of just assuming that we are using the bundled `npm`? There are a few different ways to get `npm` onto our CI machines, and additionally there are many different combos that folks might be using locally. Checking for the actual thing we care about seems much better here...